Dashboard/Client/SessionManagement/IUXClientSessionManager.cs

```diff
@@ -50,9 +50,12 @@ public interface IUXClientSessionManager
         /// <summary>
         /// Gather analytics of the users and messages.
         /// </summary>
-        SessionAnalytics GetAnalytics();
+        void GetAnalytics();

```

Comment:
Add an event for sessionAnalytics created

Reply:
This is missing ig

Reply:
Done